.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "cmake"]
 	path = cmake
-	url = git://github.com/jrl-umi3218/jrl-cmakemodules.git
+	url = https://github.com/jrl-umi3218/jrl-cmakemodules.git
```

Comment:
why not `url = ../../jrl-umi3218/jrl-cmakemodules`, as this was done in pinocchio ?

Reply:
Because it will only work if you have a similar tree of repositories as on github or gitlab. 
I think this approach is not as generic as expected.

Reply:
I will recover the previous behavior on Pinocchio.
@nim65s Do you agree with this?

Reply:
I mirrored https://github.com/jrl-umi3218/jrl-cmakemodules to https://gepgitlab.laas.fr/jrl-umi3218/jrl-cmakemodules for this exact reason :)
But it is a bit outdated now, I will push again.

Reply:
Done, does this fix your case ?

Reply:
What I meant is that if you are working with your clone of these repositories which are not located on the same root tree, then it won't work.
For instance, if you have Pinocchio on your own local gitlab server, but you don't want to also have the cmake module, then the current relative url won't work and will break the cloning.

Reply:
That's true. But I don't get why you would not want to just mirror jrl-umi3218/jrl-cmakemodules as I did on gepgitlab. If you are able to mirror stack-of-tasks/eigenpy, it is exactly the same process.

If you really want to impose one protocol, then HTTPS is probably a better choice than SSH, but it will upset users for which SSH would work and not HTTPS.

Reply:
So, let's choose `https` to cover the maximum number of users.
There is also an interesting point here: https://docs.gitlab.com/ee/ci/git_submodules.html